### DISCUSSION TITLE
X-lab-website 相关讨论记录

### DISCUSSION THREAD

**Discussion by zhicheng-ning** [X-lab2017/x-lab-website#2] [Ideas]
初步选定 [Docusaurus](https://docusaurus.io/zh-CN/) 作为开发框架，使用它搭建的案例众多：<https://docusaurus.io/showcase>，比较有名的有 [ReactNative](https://reactnative.dev/)，[Apache APISIX](https://apisix.apache.org/)，目前按照官方文档搭起一个[示例](https://tutorial.docusaurus.io/blog)

![817b458dac91d3eb3899a1b2fdbb411](https://user-images.githubusercontent.com/39022409/173310022-73f1cfea-a188-411c-9a3f-9a13e5b3cb45.png)
![e8869b5d2603c8a896aadea776f30d1](https://user-images.githubusercontent.com/39022409/173310013-fc296e9e-4976-421f-ac57-50b209bb0324.png)

**Comment by juzhiyuan** [comment]
😄 Apache APISIX 的确在使用 Docusaurus 作为网站框架，如果有任何关于 Docusaurus 使用上的问题，可以留言给我～
